src/frontend/src/app/services-health/services-health.component.ts

```diff
@@ -11,50 +11,20 @@ import { environment } from '../../environments/environment';
 export class ServicesHealthComponent implements OnInit {
   public timer: any = '';
   public service_data;
-  // public ping_data = {};
-  // public ping_info = { is_alive: false, service_status: 'service down' };
   constructor(private servicesHealthService: ServicesHealthService) { }
 
   ngOnInit() {
     this.getServiceData();
   }
 
   public getServiceData() {
-    this.servicesHealthService.getServicesData()
+    this.servicesHealthService.getAllServices()
       .subscribe(
       (data) => {
-        console.log('Service Status', data.services);
         this.service_data = data.services;
       },
       (error) => {
         console.log('error: ', error);
```

Comment:
if error then  (if could not connect; show warning message "Could not connect to Core Managment API, Make sure to set correct [core management port](/settings)" with link to navigate to settings page to fill up core port)